google_guest_agent/non_windows_accounts.go

```diff
@@ -320,6 +321,40 @@ func readGoogleUsersFile() (map[string]string, error) {
 	return res, nil
 }
 
+// Enables AuthorizedKeysCommand to be used instead of an AuthorizedKeysFile
```

Comment:
I don't think this is actually doing what this comment says. It's reading the `sshd_config` file trying to find a token `AuthorizedKeysCommand` and then rewriting the file with  	`#AuthorizedKeysCommand /usr/bin/google_authorized_keys_guest` and `#AuthorizedKeysCommandUser root` meaning you are disabling these two configurations - but erasing all the remaining configs.

Reply:
I think it's checking each existing line, and adds any line back to the file other than the existing "AuthorizedKeysCommand" and "AuthorizedKeysCommandUser" line. 

If the check was 
"if strings.Contains(line, "AuthorizedKeysCommand")" then I think it would delete all lines except those two lines. Right now the check is 
"if !strings.Contains(line, "AuthorizedKeysCommand")."

Reply:
Right, I misread the conditional statement. Thanks for the correction.